scripts/menus.php

```diff
@@ -0,0 +1,20 @@
+<?php
+function peopleMenu() {
+	$conn = require (__DIR__ . "/connection.php");
+	$query = $conn->prepare("SELECT name FROM people");
+	$query->execute();
+	$id = 1;
+	$result = $query->get_result();
+	echo "<section role='navigation'>";
+	while ( $value = mysqli_fetch_array($result) ) {
+		echo "<a href='/people.php?id=" . $id . "'><img src='/img/";
+		if (file_exists($_SERVER['DOCUMENT_ROOT'] . "/img/$value[0]_icon.png")) {
```

Comment:
`$_SERVER['DOCUMENT_ROOT']` doesn't work as intended if the files are not located in the web server's root directory.